sql/core/src/main/scala/org/apache/spark/sql/execution/joins/BroadcastNestedLoopJoinExec.scala

```diff
@@ -238,7 +250,6 @@ case class BroadcastNestedLoopJoinExec(
    *   ExistenceJoin with BuildLeft
    */
   private def defaultJoin(relation: Broadcast[Array[InternalRow]]): RDD[InternalRow] = {
-    /** All rows that either match both-way, or rows from streamed joined with nulls. */
```

Comment:
This comment is quite confusing for me, so delete here.
I am not sure how this can make sense, given e.g. `FullOuter` join in this method, will contain rows match both-way, and rows from streamed AND build side with nulls. Not only streamed side.